app/elements/submit-page.html

```diff
@@ -719,7 +713,7 @@ <h1 class='title--big hello__title fade-in'>Submit</h1>
       <div class="mdl-layout-spacer"></div>
       <div class="container mdl-cell mdl-cell--12-col">
         <p class$="section__auth-alert [[errorMessageClass]]">[[errorMessage]]</p>
-          <p class="section__description"><echo-html html$="{{startups.submit_email_text}}"></echo-html></p>
+          <p class="section__description"><echo-html html$="{{submit.country_offices.submit_email_text}}"></echo-html></p>
         <form is="iron-form" class="section__form" on-submit="submitEmail">
```

Comment:
do we need submit.counry_offices? Can't we have just one element submitCountryOffices?